sources/packages/api/src-sql/EducationProgramsOfferings/Create-education-programs-offerings.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE IF NOT EXISTS education_programs_offerings(
+  id SERIAL PRIMARY KEY,
+  offering_name VARCHAR(300) NOT NULL,
+  study_start_date Date NULL,
+  study_end_date Date NULL,
+  break_start_date Date NULL,
+  break_end_date Date NULL,
+  actual_tution_costs INT NULL,
```

Comment:
It is not clear to me (or I did not find it easily) if the costs accept cents or are integers.

Reply:
INT is ok. it may need to be changed but unknown at this point in time